### ISSUE TITLE
Hyperparameters for reproducing results for baselines

### ISSUE THREAD

**Issue by chauhan-utk** [RockySJ/WDGRL#6]
Can you provide the hyperparameters for reproducing the results for the baselines?
For example, there is one for MMD in office-caltech folder but the other methods have zero coefficients and I could not find any reference in paper.

**Comment by zhasan49** [comment]
Hi, I am also interested in this. I am trying to reproduce the results (in particular for DANN and the WDGRL) for my Thesis. I would greatly appreciate this as so far I have not been able to reproduce the results with the provided code (this is for the Office Caltech dataset).

**Comment by RockySJ** [comment]
Sorry for the late response. Office Caltech dataset is small and the result may be a little unstable and the random seed may have big effect. For each adaptation task (totally 12) in this dataset, the hyper-parameters may be a little different and actually I forgot to record the hyper-parameters used before. As for WDGRL, I suggest the hyper-parameters set as follows:

wd_param: 0.01 ~ 0.1
gp_param: 10 ~ 50
lr_wd_D: 1e-3
D_train_num: 10

The DANN method is a little hard to tune and I suggest that “grl_lambda” is set to 1 and “dann_param” set from 0.0001 to 0.001. 

If you still can not reproduce the reported result, we can have a further discussion.

**Comment by chauhan-utk** [comment]
Thanks for the response. I already tried to reproduce your results and was able to match the performance for the Office Caltech dataset.

**Comment by zhasan49** [comment]
Thank you very much for the response. @chauhan-utk Would you be willing to share your hyper-parameters?
